### DISCUSSION TITLE
Zooming in the image?

### DISCUSSION THREAD

**Discussion by haspden** [mebjas/html5-qrcode#400] [Q&A]
We have small QR codes that come out of focus when the QR code is brought close to the camera, 

Enlarging / zooming the image would be a good solution to poor cameras with poor macro focus

Some examples on a staff LFD testing app I am trying to build

![image](https://user-images.githubusercontent.com/4039041/151093017-9cf6eda5-3239-482f-8b31-a5176514c7f0.png)

**Comment by mebjas** [comment]
Planning to work on zooming in the future. Please track this feature request - https://github.com/mebjas/html5-qrcode/issues/330
